### ISSUE TITLE
Documentation for Postprocessors being skipped by Sphinx in RTD

### ISSUE THREAD

**Issue by ngoiz** [ImperialCollegeLondon/sharpy#30]
With the exception of `SaveData` and `BeamPlot`, RTD is failing to import the post processors' documentation due to the following error:

```
WARNING: autodoc: failed to import class 'postproc.asymptoticstability.AsymptoticStability' from module 'sharpy'; the following exception was raised:
cannot import name 'solvers' from 'sharpy' (/home/docs/checkouts/readthedocs.org/user_builds/ic-sharpy/checkouts/master/sharpy/__init__.py)
```

**Comment by ngoiz** [comment]
Fixed: The issue was with some modules being excluded in `exclude_patterns` in the `conf.py`. 

Will close once the pull request is done

**Comment by ngoiz** [comment]
This was fixed by #31
